src/ast/codegen_llvm.cpp

```diff
@@ -565,15 +573,17 @@ void CodegenLLVM::visit(Call &call)
     //   }
     // }
     //}
+    // clang-format off
```

Comment:
I see clang-format is making an interesting choice about this code. Can its settings be adjusted to get it to format this section how we want? I'd rather not introduce lots of "clang-format off" lines throughout the codebase so maybe we should just suck it up and accept its changes here if not (since it's not awful).

Reply:
I'll take it out, as we already have the comment above it the formatting doesn't add much.